cmd/kubeadm/app/master/apiclient.go

```diff
@@ -35,22 +35,35 @@ import (
 
 const apiCallRetryInterval = 500 * time.Millisecond
 
-func CreateClientAndWaitForAPI(adminConfig *clientcmdapi.Config) (*clientset.Clientset, error) {
+func createAPIClient(adminKubeconfig *clientcmdapi.Config) (*clientset.Clientset, error) {
 	adminClientConfig, err := clientcmd.NewDefaultClientConfig(
-		*adminConfig,
+		*adminKubeconfig,
 		&clientcmd.ConfigOverrides{},
 	).ClientConfig()
 	if err != nil {
-		return nil, fmt.Errorf("<master/apiclient> failed to create API client configuration [%v]", err)
+		return nil, fmt.Errorf("<cmd/token> failed to create API client configuration [%v]", err)
```

Comment:
These were intended to just be debug info identifying what module/file the message was from, IIRC they might actually be short lived and going away in future. This one was a bad copy paste, it should be master/apiclient still, propose I fix it to master/apiclient and we'll sort that out with output cleanup in the future. It the method gets moved to a more generic module we can update prefix then as well.